platform/ios/docs/guides/Adding Points to a Map.md

```diff
@@ -1,82 +1,58 @@
 # Adding Points to a Map
 
-Mapbox offers a few different ways to add points to a map, each with different tradeoffs.
+## **Annotations API**
 
-## MGLPointAnnotation
+Our annotations API includes the `MGLAnnotationImage`, and `MGLAnnotationView` classes. These are most similar to MapKit’s annotation class and provide a familiar interface for working with markers and callouts.
 
-It’s straightforward to add an annotation to a map. You can use `MGLPointAnnotation` as is, or you can subclass it to add annotations with richer data.
+| ![MGLAnnotationImage](img/adding-points-to-a-map/annotation-image.png "MGLAnnotationImage")               | ![MGLAnnotationView](img/adding-points-to-a-map/annotation-view.png "MGLAnnotationView")              |
+|----------------------|---------------------|
+| `MGLAnnotationImage` | `MGLAnnotationView` |
 
-```swift
-let annotation = MGLPointAnnotation()
-annotation.coordinate = CLLocationCoordinate2D(latitude: 45.5076, longitude: -122.6736)
-annotation.title = "Bobby's Coffee"
-annotation.subtitle = "Coffeeshop"
-mapView.addAnnotation(annotation)
-```
+**MGLAnnotationImage** is an annotation class that is easily customizable with any `UIImage`.
+It is highly performant, as the images are rendered directly in OpenGL. However, if you need to animate your annotations or control z-layer ordering, consider working with **MGLAnnotationView** which supports basic CALayer animations and can be stacked by using `zPosition` property on `CALayer`.
```

Comment:
> which supports basic CALayer animations 

This could be interpreted to mean that we _only_ support “basic CALayer animations” (which could incidentally also be confused with `CABasicAnimation`). `MGLAnnotationView` aims to support any animation that can be applied to a `UIView`, which could be from Core Animation/UIKit or elsewhere.